local/bin/py/update_pre_build.py

```diff
@@ -11,7 +11,7 @@
 import fileinput
 import json
 import re
-
+import os 
```

Comment:
not a blocker, but i notice we're importing the entire `os` module here, and importing large chunks of it above in two different statements. In the future it would make sense to either import the entire module and refactor the function calls to be namespaced, or continue to pull specific classes/functions out from a single call.

Reply:
indeed, I just did this as a quick fix, but at some point I will refactor the script.. I find it too slow to run. and too dependent of the file name